src/library_html5.js

```diff
@@ -1514,6 +1514,11 @@ var LibraryHTML5 = {
 #if HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS
     var canPerformRequests = JSEvents.canPerformEventHandlerRequests();
 
+    if (navigator.userActivation) {
+      // Use transient activation status instead for browsers that support it
+      canPerformRequests = navigator.userActivation.isActive;
+    }
```

Comment:
Should this just be part of canPerformEventHandlerRequests?

Reply:
Makes sense, I thought there was another stricter level than transient activation, but it seems not.
I've changed the PR accordingly.